### ISSUE TITLE
[Feature request] Ability to move recording from unspecified (clear) file location to SSD after shot

### ISSUE THREAD

**Issue by owlgreens** [mirsadm/motioncam#142]
I'm getting the fewest dropped frames when I leave the recording location clear. However, I lose the ability to offload the recording to an SSD, without first converting to DNG, while in the field. It would be great if there was a way to access the container file created when the recording location is unspecified, and be able to move it to an SSD after the shot. Alternatively, it would be great if we could specify the recording location without decreasing the write speed and increasing the chances of dropped frames. Thank you again for the incredible work!

**Comment by mirsadm** [comment]
Yep that is a good idea I need to do that. On my phone writing to the app storage is significantly faster as well, not too sure why.
